handlers.go

```diff
@@ -59,7 +59,12 @@ type lookupResponse struct {
 }
 
 func (rp *Ringpop) adminLookupHandler(ctx json.Context, req *lookupRequest) (*lookupResponse, error) {
-	return &lookupResponse{Dest: rp.Lookup(req.Key)}, nil
+	dest, err := rp.Lookup(req.Key)
+	if err != nil {
+		return nil, err
+	} else {
+		return &lookupResponse{Dest: dest}, nil
```

Comment:
> idiomatic go is

Have you got a source/reference for this?

If the method has more than two return points, then I use the pattern you described. However, if there is only two, I prefer the if/else because I think it's clearer.